### ISSUE TITLE
App crashed when loading note

### ISSUE THREAD

**Issue by jadeqwang** [keybits/permanote#2]
What it looks like

![image](https://cloud.githubusercontent.com/assets/1584904/13362341/9fad0152-dc78-11e5-84c5-1fae70924cdc.png)

Steps to repro:
- Create new Permanote grain
- Create a new note, put some stuff in it, save it
- Attempt to edit the note

Debug log: Noms

```

...** SANDSTORM SUPERVISOR: Starting up grain.
sandstorm/supervisor.c++:1165: warning: ip_tables kernel module not loaded; cannot set up transparent network forwarding.
 * Running on http://0.0.0.0:8000/ (Press CTRL+C to quit)
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET / HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET /static/css/note.min.css HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET /static/css/hilite.css HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET /static/js/jquery-1.12.0.min.js HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET /static/js/bootstrap.min.js HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET /static/js/jquery.inline-attachment.min.js HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:03] "GET /static/js/jquery.hotkeys.js HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /create/ HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /static/css/note.min.css HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /static/css/hilite.css HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /static/js/jquery-1.12.0.min.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /static/js/bootstrap.min.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /static/js/jquery.inline-attachment.min.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:06] "GET /static/js/jquery.hotkeys.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "POST /create/ HTTP/1.1" 302 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /wf-shopping-list/ HTTP/1.1" 200 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /static/css/note.min.css HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /static/css/hilite.css HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /static/js/jquery-1.12.0.min.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /static/js/bootstrap.min.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /static/js/jquery.inline-attachment.min.js HTTP/1.1" 304 -
127.0.0.1 - - [26/Feb/2016 19:00:26] "GET /static/js/jquery.hotkeys.js HTTP/1.1" 304 -
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 199, in run_wsgi
    from werkzeug.debug.tbtools import get_current_traceback
ImportError: No module named debug.tbtools
** SANDSTORM SUPERVISOR: Grain still in use; staying up for now.
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 199, in run_wsgi
    from werkzeug.debug.tbtools import get_current_traceback
ImportError: No module named debug.tbtools
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 295, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 321, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 216, in handle
    rv = BaseHTTPRequestHandler.handle(self)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 251, in handle_one_request
    return self.run_wsgi()
  File "/opt/app/env/local/lib/python2.7/site-packages/werkzeug/serving.py", line 199, in run_wsgi
    from werkzeug.debug.tbtools import get_current_traceback
ImportError: No module named debug.tbtools
*** lost connection to grain (probably because it shut down) ***
```

**Comment by keybits** [comment]
Hi Jade, thanks for the bug report.

In my haste to revert to Python 2 from Python 3, I didn't spend enough time using the app in local development so vargrant-spk didn't pick up all the files.

This should now be fixed with version 0.1.4 which I've just released and pushed to the app market.

**Comment by keybits** [comment]
I should really write some tests as my next task to avoid such things!
